### ISSUE TITLE
Missing `aria-owns` attribute

### ISSUE THREAD

**Issue by FranciscoG** [sagalbot/vue-select#757]
According to the ARIA spec, use of the `role="combobox"` requires some ARIA attributes that this component is missing. 

First it looks like you're implement the older Aria 1.0 pattern.  So a larger fix would be to update it to the 1.1 pattern (see more info in link below)

Using the 1.0 pattern (putting `role="combobox"` on the input element) you're missing the `aria-owns` attribute, and of course give the popup `<ul>` an id.  This is probably where I would let users pass in an ID prop themselves so they can handle uniqueness and you don't have to deal with it.

source: https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-6

> In a combobox implementing the ARIA 1.0 pattern:
> - The element that serves as the textbox has role combobox.
> - **When the combobox popup is visible, the element with role combobox has aria-owns set to a value that refers to an element with role listbox.**
> - the element with role combobox has a value for aria-haspopup of listbox. Note that elements with role combobox have an implicit aria-haspopup value of listbox.

**Comment by sagalbot** [comment]
Thanks for reporting this! Fixed in the v3.5.2 release.
